CZ/pyranda_rayleigh_taylor/iterative_pgen.py

```diff
@@ -0,0 +1,72 @@
+import numpy as np
+from trata.sampler import LatinHyperCubeSampler as LHS
+from trata.adaptive_sampler import ExpectedImprovementSampler as EIS
+from sklearn.gaussian_process import GaussianProcessRegressor as gpr
+from sklearn.preprocessing import MinMaxScaler as MMS
+from maestrowf.datastructures.core import ParameterGenerator
+
+
+# Generate samples iteratively
+
+
+def get_custom_generator(env, **kwargs):
+
+    # This is the range of the 2 variables
+    ls_test_box = [[0.3, 0.65], [0.85, 1.15]]
+
+    Nruns = int(kwargs.get("NRUNS", env.find("NRUNS").value))
+    Ncand = int(kwargs.get("NCAND", env.find("NCAND").value))
```

Comment:
Is this file not meant to work because I don't see Ncand in any of the yaml files.

Reply:
I created a new iterative yaml file that has the `NCAND` variable defined. It also has some changes to this iterative_pgen.py file in order for it to run successfully.